src/internal.c

```diff
@@ -36,6 +36,7 @@
 #ifdef NO_INLINE
     #include <wolfssl/wolfcrypt/misc.h>
 #else
+    #define WOLFSSL_MISC_COMPILED
```

Comment:
I believe this name is misleading.  While the file is being compiled it is NOT being compiled directly.  It is being included as if it were a header.